src/index.ts

```diff
@@ -30,6 +30,12 @@ namespace m3u8stream {
     on(event: 'progress', progress: Progress, totalSegments: number, downloadedBytes: number): this;
     on(event: string | symbol, listener: (...args: any) => void): this;
   }
+
+
+  export interface M3u8stream {
+    (playlistURL: string, options?: Options): Stream;
+    parseHumanTime(time: number | string): number;
+  }
```

Comment:
not a fan of how this ended up looking. but it looks like this is the only way to do this in typescript

do you mind lowercasing the name of this so it's consistent with the namespace name?

Reply:
sure can do,
actually uppercased it since the other interface's are uppercase too and i wasn't sure whether naming the interface the same as the namespace would produce problems